java/java.source.base/src/org/netbeans/modules/java/source/save/CasualDiff.java

```diff
@@ -1900,7 +1900,14 @@ protected int diffSwitch(JCSwitch oldT, JCSwitch newT, int[] bounds) {
         PositionEstimator est = EstimatorFactory.cases(oldT.getCases(), newT.getCases(), diffContext);
         localPointer = diffList(oldT.cases, newT.cases, localPointer, est, Measure.MEMBER, printer);
 
-        copyTo(localPointer, bounds[1]);
+        List<JCCase> cases = newT.cases;
+        if (cases.size() != 0) {
+            String caseKind = String.valueOf(CasualDiff.getCaseKind(cases.get(0)));
+            if (caseKind.equals("RULE")) {
+		printer.newline();
+            }
+        }
+	copyTo(localPointer, bounds[1]);
```

Comment:
Please use not a TAB but spaces.

Reply:
@junichi11 , done with the changes requested

Reply:
Thanks!